### ISSUE TITLE
Unable to build : [enum-conversion] error (conversion from 'GtkStateType' to 'GtkStateFlags')

### ISSUE THREAD

**Issue by kikislater** [AyatanaIndicators/ayatana-ido#2]
Hi,
Unable to build on Manjaro, any ideas ?  : 

```
make  all-recursive
make[1]: Entering directory '/home/sylvain/.cache/yay/ayatana-ido/src/ayatana-ido-0.4.90'
Making all in src
make[2]: Entering directory '/home/sylvain/.cache/yay/ayatana-ido/src/ayatana-ido-0.4.90/src'
  CC       idomediaplayermenuitem.lo
idomediaplayermenuitem.c: In function 'ido_media_player_menu_item_draw':
idomediaplayermenuitem.c:84:36: warning: 'gtk_widget_get_state' is deprecated: Use 'gtk_widget_get_state_flags' instead [-Wdeprecated-declarations]
   84 |                                    gtk_widget_get_state (widget),
      |                                    ^~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtkapplication.h:27,
                 from /usr/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from idomediaplayermenuitem.h:25,
                 from idomediaplayermenuitem.c:24:
/usr/include/gtk-3.0/gtk/gtkwidget.h:827:23: note: declared here
  827 | GtkStateType          gtk_widget_get_state              (GtkWidget    *widget);
      |                       ^~~~~~~~~~~~~~~~~~~~
idomediaplayermenuitem.c:84:36: error: implicit conversion from 'GtkStateType' to 'GtkStateFlags' [-Werror=enum-conversion]
   84 |                                    gtk_widget_get_state (widget),
      |                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
idomediaplayermenuitem.c: In function 'track_info_label_new':
idomediaplayermenuitem.c:123:3: warning: 'gtk_misc_set_alignment' is deprecated [-Wdeprecated-declarations]
  123 |   gtk_misc_set_alignment (GTK_MISC (label), 0.0, 0.5);
      |   ^~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtklabel.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaccellabel.h:35,
                 from /usr/include/gtk-3.0/gtk/gtk.h:33,
                 from idomediaplayermenuitem.h:25,
                 from idomediaplayermenuitem.c:24:
/usr/include/gtk-3.0/gtk/deprecated/gtkmisc.h:72:6: note: declared here
   72 | void gtk_misc_set_alignment (GtkMisc *misc,
      |      ^~~~~~~~~~~~~~~~~~~~~~
idomediaplayermenuitem.c:123:3: warning: 'gtk_misc_get_type' is deprecated [-Wdeprecated-declarations]
  123 |   gtk_misc_set_alignment (GTK_MISC (label), 0.0, 0.5);
      |   ^~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtklabel.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaccellabel.h:35,
                 from /usr/include/gtk-3.0/gtk/gtk.h:33,
                 from idomediaplayermenuitem.h:25,
                 from idomediaplayermenuitem.c:24:
/usr/include/gtk-3.0/gtk/deprecated/gtkmisc.h:70:9: note: declared here
   70 | GType   gtk_misc_get_type      (void) G_GNUC_CONST;
      |         ^~~~~~~~~~~~~~~~~
idomediaplayermenuitem.c: In function 'ido_media_player_menu_item_init':
idomediaplayermenuitem.c:138:3: warning: 'gtk_widget_set_margin_right' is deprecated: Use 'gtk_widget_set_margin_end' instead [-Wdeprecated-declarations]
  138 |   gtk_widget_set_margin_right (self->player_icon, 6);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtkapplication.h:27,
                 from /usr/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from idomediaplayermenuitem.h:25,
                 from idomediaplayermenuitem.c:24:
/usr/include/gtk-3.0/gtk/gtkwidget.h:1091:10: note: declared here
 1091 | void     gtk_widget_set_margin_right  (GtkWidget *widget,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
idomediaplayermenuitem.c:147:3: warning: 'gtk_widget_set_margin_right' is deprecated: Use 'gtk_widget_set_margin_end' instead [-Wdeprecated-declarations]
  147 |   gtk_widget_set_margin_right (self->album_art, 8);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtkapplication.h:27,
                 from /usr/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /usr/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from idomediaplayermenuitem.h:25,
                 from idomediaplayermenuitem.c:24:
/usr/include/gtk-3.0/gtk/gtkwidget.h:1091:10: note: declared here
 1091 | void     gtk_widget_set_margin_right  (GtkWidget *widget,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [Makefile:675: idomediaplayermenuitem.lo] Error 1
make[2]: Leaving directory '/home/sylvain/.cache/yay/ayatana-ido/src/ayatana-ido-0.4.90/src'
make[1]: *** [Makefile:537: all-recursive] Error 1
make[1]: Leaving directory '/home/sylvain/.cache/yay/ayatana-ido/src/ayatana-ido-0.4.90'
make: *** [Makefile:445: all] Error 2
```

**Comment by kikislater** [comment]
Fixed here : https://github.com/AyatanaIndicators/ayatana-ido/pull/3

**Comment by sunweaver** [comment]
Resolved by 033c9f49dbb70842094ca89017ab58faf9540bb9.
